test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -24,6 +24,10 @@
 
 namespace Microsoft.ML.Tests.TrainerEstimators
 {
+    [CollectionDefinition(nameof(NoParallelizationDefinition), DisableParallelization = true)]
+    public class NoParallelizationDefinition { }
+
+    [Collection(nameof(NoParallelizationDefinition))]
```

Comment:
Is there something specific about these LightGBM tests that differs from others?
https://github.com/search?q=repo%3Adotnet%2Fmachinelearning%20LightGBMFact&type=code

Reply:
Honestly I'm not sure. I just put this tag around the test I saw failing and it seems to have worked. So its possible there is? Or its possible it just has to do with test ordering? I'm not sure.

Reply:
It would be good to at comment that we've observed concurrency issues in LightGBM when running these in parallel.  Consider keeping an issue open to track that and link to the issue since we still don't understand why it's failing.